src/Equinox.DynamoStore/DynamoStore.fs

```diff
@@ -102,120 +135,222 @@ module Unfold =
 ///   (also Tip magic value is tipMagicI: Int32.MaxValue, not "-1")
 /// - etag is managed explicitly (on Cosmos DB, its managed by the service and named "_etag")
 [<NoEquality; NoComparison>]
-type Batch =
-    {   p : string // "{streamName}"
+type Batch = {
```

Comment:
I wonder can stroustrup be turned off for records - the other format works well because `member`s can be level with the opening brace